inc/datahandlers/user.php

```diff
@@ -868,6 +868,31 @@ function verify_language()
 	}
 
 	/**
+	 * Verifies if a style is valid for this user or not.
+	 *
+	 * @return boolean True when valid, false when invalid.
+	 */
+	function verify_style()
+	{
+		global $lang;
+
+		$user = &$this->data;
+
+		if($user['style'] && !defined('IN_ADMINCP'))
```

Comment:
The PR now changes the theme immediately on the fly if the user can't use it during a page visit, right? In that case it doesn't matter that the admin will be able to switch to a disallowed theme because it will get changed to a default theme anyways. So I'd show an error.